bin/Dockerfile.releaser

```diff
@@ -0,0 +1,14 @@
+FROM dockercore/golang-cross
```

Comment:
how is this different from cyberark/goreleaser or goreleaser/goreleaser?

I want to be mindful of using lots of different images for this across repos - my gut sense is that it'll be simpler to maintain if we can be consistent across projects. wdyt?

Reply:
This is building exactly the same thing that Secretless does. We should be building and maintaining our own image or contribute it to goreleaser. It's the case right now so each project has to build  `cyberark/goreleaser`